packages/firebase/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@next-auth/firebase-adapter",
+  "repository": "https://github.com/nextauthjs/adapters",
+  "version": "0.0.0",
+  "description": "Firebase adapter for next-auth.",
+  "keywords": [
+    "next-auth",
+    "next.js",
+    "firebase"
+  ],
+  "homepage": "https://next-auth.js.org",
+  "bugs": {
+    "url": "https://github.com/nextauthjs/adapters/issues"
+  },
+  "main": "dist/index.js",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "build": "tsc",
+    "test": "yarn build && jest"
+  },
+  "author": "Ron Houben",
+  "contributors": [
+    "Nico Domino <yo@ndo.dev>"
+  ],
```

Comment:
❤

Reply:
@ronhouben do tell us if you want your e-mail address or any page appear here!

Reply:
Sure you can put down:
ron.houben85@gmail.com